README.md

```diff
@@ -21,11 +21,20 @@ where `<version>` is the version of the ullyses-utils package you desire.
 
 ## Usage
 
-To use the utility scripts, the module need only be imported and run:
+### Utility Scripts
+This repo houses several utility scripts that are needed to creates ULLYSES HLSPs. Most are specific to this purpose, but some may be generally useful: specific functions in such scripts are described below.
+
+**`match_aliases.match_aliases`**
+
+Given an input target name, an attempt is made to match it to any aliases in the ULLYSES sample.
+
 ```
-from ullyses_utils import ullyses_config
+from match_aliases import match_aliases
+match_aliases("BAT99 105")
```

Comment:
I think I would add here the different columns that are available to specify. I think I added this in the docstring so you might be able to copy/paste